src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,128 @@
 package core.basesyntax;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationFailureException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorldTest {
+    private static final int VALID_AGE = 20;
+    private static RegistrationService registrationService;
+    private static User validUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void beforeEach() {
+        validUser = new User();
+        validUser.setLogin("ValidLogin");
+        validUser.setPassword("password");
+        validUser.setAge(VALID_AGE);
+    }
+
+    @Test
+    void register_addUser_Ok() {
+        validUser.setLogin("newLogin");
+        User result = registrationService.register(validUser);
+        assertEquals(validUser, result, "The valid user and result aren't the same.");
+    }
+
+    @Test
+    void register_passwordEdges_Ok() {
+        validUser.setLogin("Marco1");
+        validUser.setPassword("123456");
+        User result = registrationService.register(validUser);
+        assertEquals(result, validUser, "User and added result aren't the same.");
+
+        validUser = new User();
+        validUser.setLogin("Parmezan");
+        validUser.setPassword("12345678910");
+        validUser.setAge(VALID_AGE);
```

Comment:
I can't do that. Because if it's ok test, I am adding user to DataBase, and I can't add the same user to it. And if I will change some fields without creating new user I will change the user in DataBase by reference